Core/Commands/Command.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+using Citizen = CitizenFX.Core.Player;
+
+namespace IgiCore.Core.Commands
```

Comment:
Is there any reason you put this in Core?

Reply:
I noticed that was where other abstract stuff was, and wasn't sure if we would want to do any client only commands at some point